app/views/categories/show.html.erb

```diff
@@ -11,21 +11,23 @@
 <% end %>
 
 <% post_count = @posts.count %>
-<div class="has-color-tertiary-500 category-meta" title="<%= post_count %>">
-  <div>
-    <%= short_number_to_human post_count, precision: 1, significant: false %>
-    <%= 'post'.pluralize(post_count) %> &middot;
+<div class="has-color-tertiary-500 category-meta">
+  <span>
```

Comment:
This containing span is the main difference from my suggested approach. It doesn't seem to cause any problems so I see no reason not to stick with the span. (No change required here.)